runtime/compiler/compile/J9Compilation.cpp

```diff
@@ -665,11 +667,26 @@ J9::Compilation::canAllocateInline(TR::Node* node, TR_OpaqueClassBlock* &classIn
       {
       classRef      = node->getSecondChild();
 
-      // In the case of dynamic array allocation, return 0 indicating variable dynamic array allocation
+      // In the case of dynamic array allocation, return 0 indicating variable dynamic array allocation,
+      // unless value types are enabled, in which case return -1 to prevent inline allocation
       if (classRef->getOpCodeValue() != TR::loadaddr)
          {
          classInfo = NULL;
-         return 0;
+         if (areValueTypesEnabled)
+            {
+            if (self()->getOption(TR_TraceCG))
+               {
+               traceMsg(self(), "cannot inline array allocation @ node %p because value types are enabled\n", node, size);
```

Comment:
Windows build failure due to the reference to the uninitialized `size` variable in the call to `traceMsg`.

Reply:
Corrected in commit https://github.com/eclipse/openj9/pull/9139/commits/a328c0329eb9ecb0ba1648741437e5cb24d151ad